www/app/Http/Controllers/MemoryController.php

```
@@ -653,6 +737,47 @@ public function createDatabase(Request $request)
         }
     }
 
+    /**
+     * Delete a memory database (soft delete only).
+     */
+    public function deleteDatabase(Request $request, MemoryDatabase $memoryDatabase)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)